src/applications/vaos/reducers/newAppointment.js

```diff
@@ -49,6 +50,47 @@ function setupFormData(data, schema, uiSchema) {
   );
 }
 
+function updateFacilitiesSchemaAndData(systems, facilities, schema, data) {
```

Comment:
Most of the reducer work in here is taking the data from the actions and updating the page schema to reflect if we have systems and facilities and what options we need to display to users. This function is used for several actions.